### ISSUE TITLE
'CreateCustomRewardsRequest.IsEnabled' defaults to false.

### ISSUE THREAD

**Issue by Demkeys** [TwitchLib/TwitchLib#1131]
```
TwitchLib.Api.Helix.Models.ChannelPoints.CreateCustomReward.CreateCustomRewardsRequest newRewardReq = new();
newRewardReq.Title = "Created From Code 02";
newRewardReq.Cost = 1;

var newReward = await api.Helix.ChannelPoints.CreateCustomRewardsAsync(
    user_id, newRewardReq
);
```
The following code creates a custom reward, but the reward is disabled. I believe this is because the IsEnabled property (bool) defaults to false. However, the [Create Custom Rewards](https://dev.twitch.tv/docs/api/reference/#create-custom-rewards) info in the Twitch API Reference mentions that if 'is_enabled' is not specified, it defaults to true. This isn't a major issue but I feel like this can cause slight confusion.

**Comment by Hampo** [comment]
What version of `TwitchLib.Api` are you running? Looking at the [dev branch](https://github.com/TwitchLib/TwitchLib.Api/blob/dev/TwitchLib.Api.Helix.Models%2FChannelPoints%2FCreateCustomReward%2FCreateCustomRewardsRequest.cs#L30-L34), it doesn't have an initial value set

**Comment by Mahsaap** [comment]
> What version of `TwitchLib.Api` are you running? Looking at the [dev branch](https://github.com/TwitchLib/TwitchLib.Api/blob/dev/TwitchLib.Api.Helix.Models%2FChannelPoints%2FCreateCustomReward%2FCreateCustomRewardsRequest.cs#L30-L34), it doesn't have an initial value set

It's not that. We have had this in the past. If it's not set when the model is created c# sets it to false. There is a thing we need to add but I need to find it.

**Comment by Syzuna** [comment]
Yes the default value for bools that are just declared but never assigned to is false in C#

**Comment by Mahsaap** [comment]
https://github.com/TwitchLib/TwitchLib.Api/blob/8e57e964e6c1030378da16ab1b2c34f06d245a1e/TwitchLib.Api.Helix.Models/Channels/ModifyChannelInformation/ModifyChannelInformationRequest.cs#L5

Like that
